components/brave_wallet/browser/zcash/zcash_transaction.h

```diff
@@ -56,10 +56,10 @@ class ZCashTransaction {
     std::string utxo_address;
     Outpoint utxo_outpoint;
     uint64_t utxo_value = 0;
+    uint32_t n_sequence = 0xffffffff;
 
+    std::vector<uint8_t> script_pub_key;
```

Comment:
Can we just dervice script_pubkey from address like bitcoin does?

Reply:
I'd be interested to know if this is possible as well

Reply:
Yes, it is possible, i've just used librustzcash approach where they extract script_pub_key from the backend response.